assets/airports/kstl.json

```diff
@@ -1334,7 +1334,7 @@
             "origin": "kstl",
             "destination": "",
             "category": "departure",
-            "route": "KSTL.WHRLI5.MCM",
+            "route": "KSTL30L.WHRLI5.MCM",
```

Comment:
29 or 30L? Looking at previous spawnPatterns, it appears that you've used 29 for any SIDs going via DAFUS.